docs/proposals/20200330-spot-instances.md

```diff
@@ -262,6 +263,73 @@ This would be preferable as a DaemonSet would not be required on workload cluste
 Since this is not essential for running on non-guaranteed instances and existing solutions exist for each provider,
 users can deploy these existing solutions until CAPI has capacity to implement a solution.
 
+#### Termination handler design
+
+To enable graceful termination of workloads running on non-guaranteed instances,
+a termination handler pod will need to be deployed on each spot instance.
+
+The termination pod will be developed to poll the metadata service for the Node
+that it is running on.
+We will implement request/response handlers for each of the three cloud providers
```

Comment:
"each of the three cloud providers" seems a bit restrictive, maybe this can be 
```suggestion
We will implement request/response handlers for each infrastructure provider that supports non-guaranteed instances 
```